### PR TITLE
getInitialSatValue() + the model being undefined in onFunctions.js fix 

### LINKED ISSUE #484: getInitialSatValue() (ModelFunctions.js) is outdated
When the model was updated to BBM, the instances where this method is called were not updated alongside it.

### PR DESCRIPTION
**Pull request by angelac2023**
Closes #484

### CODE REVIEW

File: leaf-ui/js/onFunctions.js [hunk 1]
```diff
@@ -11,38 +11,39 @@ It also contains the setup for Rappid elements.
 $('#btn-undo').on('click', _.bind(commandManager.undo, commandManager));
 $('#btn-redo').on('click', _.bind(commandManager.redo, commandManager));
 $('#btn-clear-all').on('click', function () { clearAll() });
-// TODO: Reimplement with new backbone structure
+
```
Comment (Manarnz) [inline leaf-ui/js/onFunctions.js]: Remove extra line

File: leaf-ui/js/onFunctions.js [hunk 2]
```diff
@@ -242,59 +243,61 @@ graph.on('change:size', function (cell, size) {
 graph.on('remove', function (cell) {
     // Clear right inspector side panel
     clearInspector();
+    
+    // Leaving this here for now in case we need this later
```
Comment (Manarnz) [inline leaf-ui/js/onFunctions.js]: Maybe add a TODO for what you plan to do next with the commented code?

**Review by Manarnz — APPROVED** [review]

File: leaf-ui/js/onFunctions.js [hunk 3]
```diff
@@ -11,38 +11,39 @@ It also contains the setup for Rappid elements.
...
-        if (intention != null && intention.getInitialSatValue() != '(no value)') {
+        if (intention != null && intention.getUserEvaluationBBM(0).get('assignedEvidencePair') != '(no value)') {
             intention.removeInitialSatValue();
 
             cell.attr(".satvalue/text", "");
             cell.attr(".funcvalue/text", "");
 
-            elementInspector.$('#init-sat-value').val('(no value)');
-            elementInspector.$('.function-type').val('(no value)');
+            // Leaving this here in case we need it later
```
Comment (Domily11) [inline leaf-ui/js/onFunctions.js]: Put this as TODO maybe? Like TODO: determine if we need this part

File: leaf-ui/js/onFunctions.js [hunk 4]
```diff
@@ -11,38 +11,39 @@ It also contains the setup for Rappid elements.
...
         var cell = cellView.model;
-        var intention = model.getIntentionByID(cellView.model.attributes.nodeID);
+        var intention = cell.get('intention'); 
 
         if (intention != null) {
             intention.removeFunction();
             cell.attr(".funcvalue/text", "");
-            elementInspector.$('.function-type').val('(no value)');
+
+            // Leaving this here in case we need it later
```
Comment (Domily11) [inline leaf-ui/js/onFunctions.js]: Same as previous

File: leaf-ui/js/onFunctions.js [hunk 5]
```diff
@@ -242,59 +243,61 @@ graph.on('change:size', function (cell, size) {
...
+    //     var link = cell;
+    //     var source = link.prop("source");
+    //     var target = link.prop("target");
+
+    //     for (var i = 0; i < graph.getElements().length; i++) {
+    //         if (graph.getElements()[i].prop("id") == source["id"]) {
+    //             source = graph.getElements()[i];
+    //         }
+    //         if (graph.getElements()[i].prop("id") == target["id"]) {
+    //             target = graph.getElements()[i];
```
Comment (Domily11) [inline leaf-ui/js/onFunctions.js:292]: Add a TODO?

**Review by Domily11 — APPROVED** [review]
Finish implementing these comments and I think you are good to go! I haven't seen any bugs at the moment, but I think it might be better to clarify what's supposed to happen to the NB labels when the labels are cleared

**Review by amgrubb — APPROVED** [review]
Looks good. Go ahead and Squash and merge.

**Pull request merged**